RULES.md

```diff
@@ -308,6 +309,12 @@ Trips must be referred to at least once in `stop_times.txt`.
 
 A trip must visit more than one stop in `stop_times.txt` to be usable by passengers for boarding and alighting.
 
+<a name="E055"/>
+
+### E055 - Mismatching feed and agency language fields
+
+Files `agency.txt` and `feed_info.txt` must define matching `agency.agency_lang` and `feed_info.feed_lang`.
+
```

Comment:
The rule description here should have a full description of the logic in the use case, including how `mul` is handled. You could just quote the spec here.